app/views/idv/inherited_proofing/_verify.html.erb

```diff
@@ -0,0 +1,71 @@
+<%#
+locals:
+  pii - user's information
+  remote_identity_proofing - true if we're doing RIDP, false if we're doing IPP
+  step_url - generator for step URLs
+%>
+<% title t('inherited_proofing.headings.verify_information') %>
+
+<%= render PageHeadingComponent.new.with_content(t('inherited_proofing.headings.verify_information')) %>
+<div class='margin-top-4 margin-bottom-2'>
+  <div class="grid-row grid-gap grid-gap-2 padding-bottom-1">
+    <dl class="grid-col-fill margin-bottom-0">
+      <div>
+        <dt class="display-inline"> <%= t('idv.form.first_name') %>: </dt>
+        <dd class="display-inline margin-left-0"> <%= pii[:first_name] %> </dd>
+      </div>
+      <div>
+        <dt class="display-inline"> <%= t('idv.form.last_name') %>: </dt>
+        <dd class="display-inline margin-left-0"> <%= pii[:last_name] %> </dd>
+      </div>
+      <div>
+        <dt class="display-inline"> <%= t('idv.form.dob') %>: </dt>
+        <dd class="display-inline margin-left-0"> <%= pii[:dob] %> </dd>
+      </div>
+      <% if !remote_identity_proofing %>
```

Comment:
If we're not going to use the existing template then why would we leave this in? This should never be a remote_identity_proofing context, right?

Reply:
Ah, I missed that one! Thank you, I thought I had caught them all

Reply:
Actually it looks like we set this to `true` which is even more confusing since we are doing remote proofing.

If we end up re-using the template I'd suggest renaming this var to something like `should_display_state_id_number` to describe what it actually does. If we don't re-use the template we can def get rid of the conditional here.

Reply:
Definitely a template that got a bit confusing on our end, especially with the things that were taken out for MVP